resources/user-agents/browsers/android-webview/android-webview-4-0.json

```diff
@@ -1107,6 +1107,17 @@
             "Android_4_4"
           ]
         },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*)*applewebkit*(*khtml*like*gecko*)*Version/#MAJORVER#.#MINORVER#*Chrome*Mobile*Safari*",
+          "devices": {
+            "SM-J320F*": "Samsung SM-J320F"
+          },
+          "platforms": [
+            "Android_7_0",
```

Comment:
And then move this group where the others with the same `match` are, sorted by Android version.

Reply:
Sorry, move to where?

Reply:
After this group: https://github.com/browscap/browscap/blob/master/resources/user-agents/browsers/android-webview/android-webview-4-0.json#L70

Reply:
done